src/Factory.cc

```diff
@@ -69,23 +69,37 @@ std::vector<std::string> split(const std::string &_orig, char _delim)
 /////////////////////////////////////////////////
 /// \brief A factory class to generate protobuf messages at runtime based on
 /// their message descriptors. The location of the .desc files is expected
-/// via the IGN_DESCRIPTOR_PATH environment variable. This environment
+/// via the GZ_DESCRIPTOR_PATH environment variable. This environment
 /// variable expects paths to directories containing .desc files.
 /// Any file without the .desc extension will be ignored.
 class DynamicFactory
 {
   //////////////////////////////////////////////////
   /// \brief Constructor.
   /// The constructor will try to load all descriptors specified in the
-  /// IGN_DESCRIPTOR_PATH environment variable.
+  /// GZ_DESCRIPTOR_PATH environment variable.
   public: DynamicFactory()
   {
     // Try to get the list of paths from an environment variable.
-    const char *ignDescPaths = std::getenv("IGN_DESCRIPTOR_PATH");
+    const char *ignDescPaths = std::getenv("GZ_DESCRIPTOR_PATH");
```

Comment:
Ahh sorry I didn't notice before, but it will be easier in the future if we change this too

```suggestion
    const char *descPaths = std::getenv("GZ_DESCRIPTOR_PATH");
```